android/app/src/main/java/com/bounswe2020group3/paperlayer/login/LoginFragment.kt

```diff
@@ -6,20 +6,65 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
-import androidx.navigation.Navigation
+import android.widget.EditText
 import com.bounswe2020group3.paperlayer.R
 
-class LoginFragment : Fragment() {
+class LoginFragment : Fragment(),LoginContract.View {
+
+    //My presenter object
+    private lateinit var mPresenter: LoginPresenter
+    //My view
+    private lateinit var mView: View
+
+    //My UI elements
+    private lateinit var loginButton: Button
+    private lateinit var registerButton: Button
+    private lateinit var guestButton: Button
+    private lateinit var mailEditText: EditText
+    private lateinit var passwordEditText: EditText
+
+
+    /*
+    * Creates LoginPresenter Object and setView
+    * */
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        this.mPresenter=LoginPresenter()
+    }
+
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         // Inflate the layout for this fragment
         val view = inflater.inflate(R.layout.fragment_login, container, false)
-        view.findViewById<Button>(R.id.registerButton).setOnClickListener {
-            Navigation.findNavController(view).navigate(R.id.navigateToRegister)
-        }
+        this.mView=view
```

Comment:
- I think we shouldnt use variable names with m in the start as a general use. Because variable names starting with m used in native libraries. Can we rename mPresenter as presenter, mView as view for this instance?